### ISSUE TITLE
Two return?

### ISSUE THREAD

**Issue by xiaocao666tzh** [eloquentarduino/EloquentArduino#25]
In this program https://github.com/eloquentarduino/EloquentArduino/blob/master/examples/MicromlWakeWordIdentificationExample/MicromlWakeWordIdentificationExample.ino line 49 . You wrote two return? I cannot understand it! Arduino will excuate the second return?

**Comment by eloquentarduino** [comment]
First of all, in general, when there are two returns (which is pretty for sure a typo), any programming language will return the first. The second return simply gets ignored.

For the specific of the example you linked to: I read from a tutorial you want to center mic readings around 0 (so from -512 to 512) instead of the usual range of 0 to 1024. I didn't see any evidence of this in my experience, so replaced it. Just forgot to delete that line.

Bottom line: delete / ignore the second return.
